graph-proxy/.cargo/config.toml

```
@@ -0,0 +1,2 @@
+[env]
+RUST_MIN_STACK = "4194304"
```

Would be nice to have a human readable size here

```suggestion
RUST_MIN_STACK = "4194304" # 4MiB
```